single-line-diagram-core/src/main/resources/baseVoltageConstantColors.css

```diff
@@ -1,5 +1,12 @@
+/* ----------------------------------------------------------------------- */
+/* File: baseVoltageConstantColors.css ----------------------------------- */
+/* Stroke none & fill: black */
+.sld-label {stroke: none; fill: black}
 .sld-node {stroke: none; fill: black}
```

Comment:
I think all style related to components should be in components.css. There won't be a lot left then here.